BOILERPLATE_README.fr.md

```diff
@@ -24,7 +24,7 @@
 
 - Node.js (`>= 12.15, < 13.0`)
 - NPM (`>= 6.13, < 7.0`)
-- Elixir (`~> 1.10`)
+- Elixir (`~> 1.11`)
 - Erlang (`~> 23.1`)
```

Comment:
If we have `.tool-versions` now, we don’t need to include them in `BOILERPLATE_README.fr.md` and `BOILERPLATE_README.md` 🎉

Reply:
🧹

Reply:
Should we include a reference to `.tool-versions` instead?

Reply:
Yes, absolutely! Something like:

```markdown
## 🚧 Dépendances

Toutes les versions des dépendances _runtime_ sont définies dans le fichier `.tool-versions`. Ces dépendances externes sont également requises :

- PostgreSQL (`~> 12.0`)
```

```markdown
## 🚧 Dependencies

All runtime dependency versions are defined in the `.tool-versions` file. These external dependencies are also required:

- PostgreSQL (`~> 12.0`)
```

What do you think?